census/core.py

```diff
@@ -349,8 +349,20 @@ def state_county_blockgroup(self, fields, state_fips, county_fips,
 
     @supported_years(2018, 2017, 2016, 2015, 2014, 2013, 2012, 2011)
     def zipcode(self, fields, zcta, **kwargs):
```

Comment:
Unsure if deprecation handling is sufficient. Currently if `state_fips` is passed in `kwargs`the function can still be carried out; otherwise it will fail with `KeyError`